diff --git a/node_modules/coinselect/index.d.ts b/node_modules/coinselect/index.d.ts
new file mode 100644
index 0000000..4a3a5f3
--- /dev/null
+++ b/node_modules/coinselect/index.d.ts
@@ -0,0 +1,20 @@
+export interface UTXO {
+    txid: string | Buffer,
+    vout: number,
+    value: number,
+    nonWitnessUtxo? : Buffer,
+    witnessUtxo? : {
+        script: Buffer,
+        value: number
+    }
+}
+export interface Target {
+    address: string,
+    value?: number
+}
+export interface SelectedUTXO {
+    inputs?: UTXO[],
+    outputs?: Target[],
+    fee: number
+}
+export default function coinSelect(utxos: UTXO[], outputs: Target[], feeRate: number): SelectedUTXO;
