diff --git a/node_modules/hemi-viem/_esm/contracts/bitcoin-tunnel-manager.js b/node_modules/hemi-viem/_esm/contracts/bitcoin-tunnel-manager.js
index 4909558..dddad9e 100644
--- a/node_modules/hemi-viem/_esm/contracts/bitcoin-tunnel-manager.js
+++ b/node_modules/hemi-viem/_esm/contracts/bitcoin-tunnel-manager.js
@@ -1,6 +1,6 @@
 import { hemiSepolia } from "../chains/hemi-sepolia.js";
 export const bitcoinTunnelManagerAddresses = {
-    [hemiSepolia.id]: "0xBB3c3E0D5D2Ff65492deAeA721b331851befc267",
+    [hemiSepolia.id]: "0x56f1CB230e1235C2d77aF3C77Bd3E8A4c640473a", //devnet
 };
 export const bitcoinTunnelManagerAbi = [
     {
diff --git a/node_modules/hemi-viem/src/contracts/bitcoin-tunnel-manager.ts b/node_modules/hemi-viem/src/contracts/bitcoin-tunnel-manager.ts
index b76e2ab..183bf82 100644
--- a/node_modules/hemi-viem/src/contracts/bitcoin-tunnel-manager.ts
+++ b/node_modules/hemi-viem/src/contracts/bitcoin-tunnel-manager.ts
@@ -2,7 +2,7 @@ import { type Address, type Chain } from "viem";
 import { hemiSepolia } from "../chains/hemi-sepolia.js";
 
 export const bitcoinTunnelManagerAddresses: Record<Chain["id"], Address> = {
-  [hemiSepolia.id]: "0xBB3c3E0D5D2Ff65492deAeA721b331851befc267",
+  [hemiSepolia.id]: "0x56f1CB230e1235C2d77aF3C77Bd3E8A4c640473a", //devnet
 };
 
 export const bitcoinTunnelManagerAbi = [
