diff --git a/_cjs/chains/definitions/mainnet.js b/_cjs/chains/definitions/mainnet.js
--- a/_cjs/chains/definitions/mainnet.js
+++ b/_cjs/chains/definitions/mainnet.js
@@ -10,3 +10,3 @@
         default: {
-            http: ['https://eth.merkle.io'],
+            http: ['https://eth.drpc.org'],
         },
diff --git a/_cjs/chains/definitions/sepolia.js b/_cjs/chains/definitions/sepolia.js
--- a/_cjs/chains/definitions/sepolia.js
+++ b/_cjs/chains/definitions/sepolia.js
@@ -7,3 +7,3 @@
     name: 'Sepolia',
-    nativeCurrency: { name: 'Sepolia Ether', symbol: 'ETH', decimals: 18 },
+    nativeCurrency: { name: 'Ether', symbol: 'ETH', decimals: 18 },
     rpcUrls: {
diff --git a/_esm/chains/definitions/mainnet.js b/_esm/chains/definitions/mainnet.js
--- a/_esm/chains/definitions/mainnet.js
+++ b/_esm/chains/definitions/mainnet.js
@@ -7,3 +7,3 @@
         default: {
-            http: ['https://eth.merkle.io'],
+            http: ['https://eth.drpc.org'],
         },
diff --git a/_esm/chains/definitions/sepolia.js b/_esm/chains/definitions/sepolia.js
--- a/_esm/chains/definitions/sepolia.js
+++ b/_esm/chains/definitions/sepolia.js
@@ -4,3 +4,3 @@
     name: 'Sepolia',
-    nativeCurrency: { name: 'Sepolia Ether', symbol: 'ETH', decimals: 18 },
+    nativeCurrency: { name: 'Ether', symbol: 'ETH', decimals: 18 },
     rpcUrls: {
diff --git a/chains/definitions/mainnet.ts b/chains/definitions/mainnet.ts
--- a/chains/definitions/mainnet.ts
+++ b/chains/definitions/mainnet.ts
@@ -8,3 +8,3 @@
     default: {
-      http: ['https://eth.merkle.io'],
+      http: ['https://eth.drpc.org'],
     },
diff --git a/chains/definitions/sepolia.ts b/chains/definitions/sepolia.ts
--- a/chains/definitions/sepolia.ts
+++ b/chains/definitions/sepolia.ts
@@ -5,3 +5,3 @@
   name: 'Sepolia',
-  nativeCurrency: { name: 'Sepolia Ether', symbol: 'ETH', decimals: 18 },
+  nativeCurrency: { name: 'Ether', symbol: 'ETH', decimals: 18 },
   rpcUrls: {
