diff --git a/node_modules/wallet-watch-asset/src/index.js b/node_modules/wallet-watch-asset/src/index.js
index fc22949..97ab4ed 100644
--- a/node_modules/wallet-watch-asset/src/index.js
+++ b/node_modules/wallet-watch-asset/src/index.js
@@ -28,10 +28,11 @@
 async function watchAsset(provider, account, token, storage) {
   const { address, chainId, decimals, logoURI, symbol } = token;
 
-  // The only wallet known so far with EIP-747 support is MetaMask.
-  if (!provider.isMetaMask) {
-    throw new Error("Wallet not supported");
-  }
+  // The portal uses Viem Transport's, so we must skip this check
+  // for this package to work
+  // if (!provider.isMetaMask) {
+  //   throw new Error("Wallet not supported");
+  // }
 
   const storageKey = `watchedAssets:${account}`;
   const watchedAssets = storage?.getItem(storageKey) || "";
